### ISSUE TITLE
Saved Searches aren't shown

### ISSUE THREAD

**Issue by LisaSnd** [ArasLabs/toc-search-bar#7]
Is it possible that saved searches are shown, too?
So that users can click on them and get the results he needs?

![2019-01-08_11h33_12](https://user-images.githubusercontent.com/24222854/50825391-36810700-1339-11e9-97a4-ac52da5e70d9.png)

**Comment by ulricma** [comment]
That would be nice indeed!

**Comment by cgillis-aras** [comment]
Hello,

In a recent test of the SP15 version of this project, it does appear that Saved Searches are included in the results from the search bar. However, only Saved Searches that match the entered criteria are shown.

For example, I have four Saved Searches underneath Part in my environment: Buy Parts, In Work Parts, Make Parts, and Top-level Assemblies.
![image](https://user-images.githubusercontent.com/31855114/53503099-6d43e580-3a7d-11e9-974d-f5f11d3211b1.png)

If I use the criteria "Part" in the search bar, only three of these Saved Searches are returned: Buy Parts, In Work Parts, and Make Parts.
![image](https://user-images.githubusercontent.com/31855114/53503240-ae3bfa00-3a7d-11e9-8c80-699056d5a730.png)

Similarly, only Top-level Assemblies is returned if I search for "Assemblies".
![image](https://user-images.githubusercontent.com/31855114/53503277-beec7000-3a7d-11e9-8584-6993359114f9.png)

As a developer, I think this feels right, but the end user experience is more important. Would it make more sense for your users if the search criteria applied only to the ItemTypes in the TOC and all Saved Searches were returned for those ItemTypes regardless of the criteria?

Chris

**Comment by ulricma** [comment]
Yes, as a developer, naturally the filter should work as you describe. But from a users perspective, I think they would expect to see all Saved Searches independently from the filter entered. So the saved searches should be excluded from the filtering and be displayed if their parent is visible. 

I hope this helps.
